blocks/init/assets/styles/parts/utils/_colors.scss

```diff
@@ -6,3 +6,8 @@ All variables are available here:
 https://github.com/infinum/eightshift-frontend-libs/blob/develop/styles/scss/_colors.scss
 */
 
+$colors: map-get-strict($global-variables, colors);
+
+$base-primary-color: map-get-strict($colors, primary);
+$base-black-color: map-get-strict($colors, black);
+$base-white-color: map-get-strict($colors, white);
```

Comment:
Instead of defining things two times like this (and in `_shared-variables.scss`) should we maybe rather just use some kind of function for this?

Something like:

```scss
.whatever {
  color: get-from-global-settings(colors, primary);
  // or ...
  padding: get-from-global-settings(gutters);
}
```

Or in in the top map of a component:

```scss
$component: (
  colors: (
    background: get-from-global-settings(colors, black),
  ),
);
```

That way we don't need to repeat ourselves defining variables in both `manifest.json` and in `scss`

Reply:
I think this would do the trick (haven't tested tho):

```scss

@function global-settings($keys...) {
  @if variable-exists($global_settings) {    
    $value: $global_settings;
  
    @each $key in $keys {
      $value: map-get($value, $key);
    }
  
    @return $value;
  }

  @else {
    @error 'ERROR: $global_settings variable doesnt exist';
  }

}
```